addons/sale_coupon/report/sale_coupon_report_templates.xml

```diff
@@ -21,22 +21,24 @@
                                     </h1>
                                     <t t-if="o.program_id.discount_apply_on == 'specific_products'">
                                         <t t-if="len(o.program_id.discount_specific_product_ids) > 1">
-                                            <t t-set="text" t-value="'OFF ON SOME PRODUCTS*'"/>
+                                            <t t-set="text">OFF ON SOME PRODUCTS*</t>
                                             <t t-set="display_specific_products" t-value="True"/>
                                         </t>
                                         <t t-else="">
-                                            <t t-set="text" t-value="'OFF ON %s' % o.program_id.discount_specific_product_ids.name.upper()"/>
+                                            <t t-set="trad_text">OFF ON %s</t>
+                                            <t t-set="text" t-value="trad_text % o.program_id.discount_specific_product_ids.name.upper()"/>
                                         </t>
                                     </t>
                                     <t t-if="o.program_id.discount_apply_on == 'cheapest_product'">
-                                        <t t-set="text" t-value="'%s' % ('OFF ON THE CHEAPEST PRODUCT')"/>
+                                        <t t-set="text">OFF ON THE CHEAPEST PRODUCT</t>
```

Comment:
was this one incorrect to startwith ? or missing a name to insert ?

Reply:
Yes, it was not translated/translatable (+ weird to to use %s just for a simple single string)